rskj-core/src/main/java/co/rsk/peg/federation/FederationStorageProvider.java

```diff
@@ -39,5 +40,8 @@ public interface FederationStorageProvider {
     Optional<Script> getLastRetiredFederationP2SHScript(ActivationConfig.ForBlock activations);
     void setLastRetiredFederationP2SHScript(Script lastRetiredFederationP2SHScript);
 
+    void setFundTransactionUnsignedHash(Sha256Hash hash);
```

Comment:
`setSvpFundTransactionUnsignedHash`? to be consistent with storage key

Reply:
(same in the other places)

Reply:
Updated.